schemas/appeal-s78.schema.json

```diff
@@ -0,0 +1,1014 @@
+          },
+          "neighbouringSiteAddressTown": {
+            "type": "string",
+            "examples": ["Maidstone"],
+            "description": "Town / City of the site address"
+          },
+          "neighbouringSiteAddressCounty": {
+            "type": ["string", "null"],
+            "examples": ["Kent"],
+            "description": "County of the site address"
+          },
+          "neighbouringSiteAddressPostcode": {
+            "type": "string",
+            "examples": ["MD21 5XY"],
+            "description": "Postal code of the site address"
+          },
+          "neighbouringSiteAccessDetails": {
+            "type": ["string", "null"],
+            "examples": ["Ongoing road works at this address"],
+            "description": "Provided information on site accessibility on this address"
+          },
+          "neighbouringSiteSafetyDetails": {
+            "type": ["string", "null"],
+            "examples": ["Ongoing road works at this address"],
+            "description": "Provided information on site health and safety on this address"
+          }
+        },
+        "required": [
+          "neighbouringSiteAddressLine1",
+          "neighbouringSiteAddressLine2",
+          "neighbouringSiteAddressTown",
+          "neighbouringSiteAddressCounty",
+          "neighbouringSiteAddressPostcode",
+          "neighbouringSiteAccessDetails",
+          "neighbouringSiteSafetyDetails"
+        ]
+      },
+      "description": "A list of neighbouring site addresses"
+    },
+    "affectedListedBuildingNumbers": {
+      "type": ["array", "null"],
+      "items": {
+        "type": "string",
+        "examples": ["10023", "17824"]
+      },
+      "description": "A list of affected listed building IDs from Historic England"
+    },
+    "appellantCostsAppliedFor": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the appellant has applied for costs"
+    },
+    "lpaCostsAppliedFor": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the appellant has applied for costs"
+    },
+    "agriculturalHolding": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the site is an agricultural holding",
+      "$comment": "S78"
+    },
+    "tenantAgriculturalHolding": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the site is a tenant agricultural holding",
+      "$comment": "S78"
+    },
+    "otherTenantsAgriculturalHolding": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if the site is an agricultural holding with other tenants",
+      "$comment": "S78"
+    },
+    "informedTenantsAgriculturalHolding": {
+      "type": ["boolean", "null"],
+      "description": "If the site is an agricultural holding, indicates tenants are informed",
+      "$comment": "S78"
+    },
+    "appellantProcedurePreference": {
+      "type": ["string", "null"],
+      "examples": ["written", "hearing", "inquiry"],
```

Comment:
Is this not an enum? (would also apply to `lpaProcedurePreference`)

Reply:
to be honest I wasn't sure if the information is captured using the `procedureType` enum... need to double check (same for the lpa one)

Reply:
I've applied the procedureType enum settings.

Reply:
would it not make sense to reuse the APPEAL_CASE_PROCEDURE enum across all 3?